plat/common/include/pci/pci_bus.h

```diff
@@ -207,31 +207,35 @@ static struct pci_bus_handler ph __unused;
 #define PCI_ENABLE_BIT              (1u << 31)
 
 #define PCI_CONF_CLASS_ID          (0x08)
-#define PCI_CONF_CLASS_ID_SHFT     (16)
-#define PCI_CONF_CLASS_ID_MASK     (0xFF00)
+#define PCI_CONF_CLASS_ID_SHFT     (24)
+#define PCI_CONF_CLASS_ID_MASK     (0xFF)
```

Comment:
I think this is a breaking change and all PCI drivers that match on the class would have to adjust for the shifted class value.